### ISSUE TITLE
where get factors for request?

### ISSUE THREAD

**Issue by gl00m** [nicehash/simplemultialgo#2]
```
double factors[] = { 1, 7, 6 };
size_t index = nicehash_simplemultialgo_get(anames, factors, 3, &port_out);
```
where should I get these "factors[]"? what are they? I read, that it's a speed, but what speed: something common in all blockchain or my own rig's speed? And how to get it in both cases?

**Comment by nicehashdev** [comment]
Yes, it can be speed. It is only important that you provide ratios between algorithms.

**Comment by sjeatdsi** [comment]
Sorry, I don't get it. Where do I get factors for the algorithms? I can't get ratios unless I know what kind of data I'm getting in the "payout" field.

**Comment by igorvoltaic** [comment]
> Sorry, I don't get it. Where do I get factors for the algorithms? I can't get ratios unless I know what kind of data I'm getting in the "payout" field.

@sjeatdsi speed difference between desired algos
